datalad/crawler/pipelines/simple_with_stanford_lib.py

```diff
@@ -0,0 +1,49 @@
+from ..nodes.matches import xpath_match
+from ..nodes.crawl_url import crawl_url
+try:
+    from scrapy.http import Response
+    from scrapy.http import XmlResponse
+    from scrapy.selector import Selector
+
+except ImportError:  # pragma: no cover
+    class Selector(object):
+        xpath = css = None
+    Response = None
+
+
+# copied the simple_with_archives template in order to show
+def pipeline(doc_id=None,
+             x_pathmatch_="//file/@id",
+             leading_dirs_depth=1):
+
+    crawler = crawl_url("https://purl.stanford.edu/" + doc_id + ".xml")
+    # adding print_xml to incoming pipeline
+    incoming_pipeline = [  # Download all the archives found on the project page
+        crawler,
+        xpath_match(x_pathmatch_, min_count=1), # changed h-ref to xpath_match
+        configure_url,
+        print_xml
+    ]
+
+    from .simple_with_archives import pipeline as sa_pipeline
+    return sa_pipeline(leading_dirs_depth=leading_dirs_depth, incoming_pipeline=incoming_pipeline)
+
+
+def configure_url(data, keys=['url'], match='match'):
```

Comment:
please add a docstring describing what is doing and a dedicated unittest (initiate tests/test_simple_with_stanford_lib.py).